devbox.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packages": [
+    "bash@latest",
+    "clusterctl@latest",
+    "envsubst@latest",
+    "ginkgo@latest",
+    "gnumake@latest",
+    "go@latest",
+    "gojq@latest",
+    "goreleaser@latest",
+    "gotestsum@latest",
+    "kind@latest",
+    "ko@latest",
+    "kubernetes-code-generator@latest",
+    "kubernetes-controller-tools@latest",
+    "kustomize@latest",
+    "path:./hack/flakes#go-apidiff",
+    "path:./hack/flakes#go-mod-upgrade",
+    "path:./hack/flakes#golangci-lint",
+    "path:./hack/flakes#setup-envtest"
```

Comment:
can we ensure the versions here match the versions that were specified in the makefile?

Reply:
Good observation.

It is not that rare to have problems from unexpected version changes.